src/meson.build

```diff
@@ -149,12 +149,36 @@ fwupdoffline = executable(
 )
 endif
 
-resources_src = gnome.compile_resources(
-  'fwupd-resources',
-  'fwupd.gresource.xml',
-  source_dir: '.',
-  c_name: 'fu'
-)
+# this requires a new enough meson version
+if get_option('gresource_quirks').require(meson.version().version_compare('>=0.63.0'),
+    error_message: 'meson >= 0.63.0 is needed for -Dgresource_quirks=enabled').allowed()
+  fwupd_gresource_xml = custom_target('fwupd-resources-xml',
+    input : [
+      'org.freedesktop.fwupd.xml',
```

Comment:
why does this need to be compiled in?  It's installed either way isn't it?

Reply:
ya, this was trying to make it work for older meson versions -- it didn't work properly and we now have the meson check so i ripped it out.